swap/src/alice/negotiate.rs

```diff
@@ -10,6 +10,7 @@ use futures::{
 };
 use libp2p::request_response::ResponseChannel;
```

Comment:
Why the file name change? this does more than just negotiation.

Reply:
Does it? I think negotiate is more accurate than execute. We can rename it again, but I think it is not the execution.

Reply:
It provides an abstraction that allows to do one function call per step for the protocol execution.

Look at the functions present in the file, they are not related to "negotiation":

- `wait_for_locked_bitcoin`
- `lock_xmr`
- etc

would `steps.rs` make more sense to you?